package.json

```diff
@@ -23,8 +23,11 @@
     "test": "yarn jest",
     "test:watch": "yarn test --watch",
     "prebuild": "yarn test --ci && shx rm -rf dist",
-    "build": "NODE_ENV=production yarn build:js && yarn build:sass",
-    "build:js": "babel src --config-file ./config/babel.config.js --out-dir dist/js --only 'src/**/*.js' --ignore 'src/jest,src/cosmos,**/tests/**,**/fixtures/**'",
+    "build": "yarn build:babel && yarn build:sass",
+    "build:babel": "concurrently \"yarn build:babel:esm && yarn build:babel:umd\" \"yarn build:babel:cjs\"",
+    "build:babel:cjs": "babel --config-file ./config/babel.config.js --source-maps --extensions \".js,.ts,.tsx\" src --out-dir dist/js --presets=@babel/env",
```

Comment:
Can we remove the test-related code from the resulting `dist`?

Reply:
+1 that could decrease a size a bit

Reply:
Fixed. It's rather silly babel didn't respect my `ignore: []` in the `babel.config.js` :(